Source/Core/VideoCommon/DriverDetails.cpp

```diff
@@ -103,7 +103,10 @@ static BugInfo m_known_bugs[] = {
     {API_OPENGL, OS_LINUX, VENDOR_MESA, DRIVER_NOUVEAU, Family::UNKNOWN,
      BUG_SHARED_CONTEXT_SHADER_COMPILATION, -1.0, -1.0, true},
     {API_VULKAN, OS_ALL, VENDOR_NVIDIA, DRIVER_NVIDIA, Family::UNKNOWN,
-     BUG_BROKEN_MSAA_VKCMDCLEARATTACHMENTS, -1.0, -1.0, true}};
+     BUG_BROKEN_MSAA_VKCMDCLEARATTACHMENTS, -1.0, -1.0, true},
+    {API_VULKAN, OS_ALL, VENDOR_IMGTEC, DRIVER_IMGTEC, Family::UNKNOWN,
+     BUG_BROKEN_CLEAR_LOADOP_RENDERPASS, -1.0, -1.0, true},
```

Comment:
If you know the start/end versions, you can fill out the -1.0 fields (end-at version in particular).

Reply:
Currently, the ImgTec driver's version string isn't parsed in OGL, and the vk 'driverVersion' ends up following a completely different versioning system to the software releases.